homeassistant/components/epson/strings.json

```diff
@@ -0,0 +1,32 @@
+{
+  "title": "Epson Projector",
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "name": "Name",
+          "port": "[%key:common::config_flow::data::port%]",
+          "ssl": "Use SSL"
```

Comment:
Is it ever possible to have an epson projector with valid SSL ?

Reply:
If not, remove this option

Reply:
I'm pretty sure some people can use self signed certificate.
Android app has such option, but I couldn't get it working with my projector

Reply:
But then you also need to host it on a domain ? Maybe better to just drop it for now.